# Copyright (C) 2018-2020 LEIDOS.
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may not
# use this file except in compliance with the License. You may obtain a copy of
# the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations under
# the License.

# Double: The rate at which the Health Monitor Node will process message
# Units: hz
spin_rate_hz: 10.0

# Double: The timeout threshold for essential drivers
# Units: ms
required_driver_timeout: 3000.0

# Double: The time allocated for system startup
# Units: s
startup_duration: 30.0

# List of String: Required plugins for the platform to be functional
required_plugins:
  - RouteFollowing
  - InLaneCruisingPlugin
  - StopAndWaitPlugin
  - Pure Pursuit

# String: Plugin service name prefix
plugin_service_prefix: /guidance/plugins/

# String: Strategic plugin service suffix
strategic_plugin_service_suffix: /plan_maneuvers

# String: Tactical plugin service suffix
tactical_plugin_service_suffix: /plan_trajectory
